.gitignore

```diff
@@ -1,4 +1,3 @@
 /target
 **/*.rs.bk
-Cargo.lock
```

Comment:
What's the reason for committing the lockfile?

Reply:
https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html